src/main/java/net/glowstone/entity/GlowPlayer.java

```diff
@@ -921,6 +937,20 @@ public void pulse() {
                     .mapToInt(Entity::getEntityId).toArray()));
         }
         getAttributeManager().sendMessages(session);
+
+        GlowFishingHook hook = currentFishingHook.get();
+        if (hook != null) {
+            // The line will disappear if the player wanders more than 32 blocks away from the
+            // bobber, or if the player stops holding a fishing rod.
+            if (getInventory().getItemInMainHand().getType() != Material.FISHING_ROD
+                    && getInventory().getItemInOffHand().getType() != Material.FISHING_ROD) {
+                setCurrentFishingHook(null);
+            }
+
+            if (hook.location.distanceSquared(location) > 32 * 32) {
```

Comment:
Create a constant for `32` (`HOOK_MAX_DISTANCE`)

Reply:
Done.